pkg/controller/periodicity/periodicity_controller.go

```diff
@@ -0,0 +1,96 @@
+package periodicity
+
+import (
+	"time"
+
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+	informers "github.com/pingcap/tidb-operator/pkg/client/informers/externalversions"
+	v1alpha1listers "github.com/pingcap/tidb-operator/pkg/client/listers/pingcap/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/controller"
+	"github.com/pingcap/tidb-operator/pkg/label"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	kubeinformers "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	eventv1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	appslisters "k8s.io/client-go/listers/apps/v1"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog"
+)
+
+type Controller struct {
+	stsLister          appslisters.StatefulSetLister
+	tcLister           v1alpha1listers.TidbClusterLister
+	statefulSetControl controller.StatefulSetControlInterface
+}
+
+func NewController(
+	kubeCli kubernetes.Interface,
+	informerFactory informers.SharedInformerFactory,
+	kubeInformerFactory kubeinformers.SharedInformerFactory) *Controller {
+
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartLogging(klog.Infof)
+	eventBroadcaster.StartRecordingToSink(&eventv1.EventSinkImpl{
+		Interface: eventv1.New(kubeCli.CoreV1().RESTClient()).Events("")})
+	recorder := eventBroadcaster.NewRecorder(v1alpha1.Scheme, corev1.EventSource{Component: "periodiciy-controller"})
+	stsLister := kubeInformerFactory.Apps().V1().StatefulSets().Lister()
+
+	return &Controller{
+		tcLister:           informerFactory.Pingcap().V1alpha1().TidbClusters().Lister(),
+		statefulSetControl: controller.NewRealStatefuSetControl(kubeCli, stsLister, recorder),
+		stsLister:          stsLister,
+	}
+
+}
+
+func (c *Controller) Run() {
+	klog.Infof("Start to running periodicity job")
+	var errs []error
+	if controller.PodWebhookEnabled {
+		if err := c.syncStatefulSetTimeStamp(); err != nil {
+			errs = append(errs, err)
+		}
+	}
+	klog.Errorf("error happened in periodicity controller,err:%v", errors.NewAggregate(errs))
```

Comment:
```suggestion
func (c *Controller) Run(stopCh <-chan struct{}) {
    klog.Infof("Staring periodicity controller")
    defer klog.Infof("Shutting down periodicity controller")
    wait.Until(c.syncStatefulSetTimeStamp, time.Minute, stopCh)
```

run our sync function every period